File: algo-competitions/starter128-div3/XOR_And_Multiply.cpp
#include <iostream>
using namespace std;

int main()
{
    int t;
    cin >> t;
    for (; t > 0; t--)
    {
        int x;
        cin >> x;
        int b = x;
        int mxor = x ^ b;
        int mand = x & b;
        for (; (mxor % x) or (mand % x) or mxor <= 0 or mand <= 0;)
        {
            b += x;
            mxor = x ^ b;
            mand = x & b;
        }
        cout << x << " " << b << "\n";
    }
}

File: algo-competitions/starter128-div3/XOR_And_Multiply.py
t = int(input())
for _ in range(t):
    x = int(input())
    b = x
    while True:
        mxor = x & b
        mand = x ^ b
        if mxor % x == 0 and mand % x == 0 and mxor > 0 and mand > 0:
            break
        else:
            b += 2 * x

    print(x, b)
